pykern/pkcli/github.py

```diff
@@ -28,12 +28,13 @@
 _GITHUB_HOST = "github.com"
 _GITHUB_URI = "https://" + _GITHUB_HOST
 _GITHUB_API = "https://api." + _GITHUB_HOST
-_WIKI_ERROR_OK = (
-    r"fatal: remote error: access denied or repository not exported: .*wiki.git"
-)
+_WIKI_ERRORS_OK = [
+    (r"fatal: remote error: access denied or repository not exported: .*wiki.git"),
```

Comment:
Do we still need this one? See my previous comment about it.

Reply:
Missed that in your comment.

It was protected originally, first thought was to leave it since the error also returns if the repo is private and the user doesn't have access, in test_mode there is no default configured user.

However testing is skipped in the suite because of the rate limit and we log in with our users when manually testing so I'm going to remove it.

Reply:
Sounds good. My thinking is: This code runs for backups. I think the expectation is the backup user has access to everything they want to backup. So, in the case where the backup user doesn't have access we want to see (not smother) that error. In dev/test same thing goes. If a user tried to backup something they don't have access to they should know.